src/client/components/Carousel/Carousel.component.js

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react';
+import './Carousel.style.css';
+
+const Carousel = ({ imageArray }) => {
+  const show = 3;
+  const [current, setCurrent] = useState(0);
+  const imageArraLength = imageArray.length;
```

Comment:
Small thing. I would suggest renaming this variable (in all places it appear in the file). Looks like a typo?

```suggestion
  const imageArrayLength = imageArray.length;
```